imports/plugins/included/payments-example/client/settings/components/exampleSettingsForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component, PropTypes } from "react";
+import { FieldGroup } from "/imports/plugins/core/ui/client/components";
+import { Button } from "react-bootstrap";
```

Comment:
Avoid using `react-bootstrap`. We need to remove it because it duplicates our own components. (Namely in the `core/email` plugin.

We have a button component that in `/imports/plugins/core/ui/client/components` that you can use.